trulens_eval/examples/expositional/models/hugging__Face_.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>since this notebook is focused on the Vectara HHEM evaluator, can you rename the notebook to reflect that?</p><p> </p><p>Suggestion: <code>vectara_hallucination_evaluator.ipynb</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/950/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>